optimism-mainnet.json

```diff
@@ -612,7 +612,6 @@
         "name": "Kyber Network Crystal v2",
         "symbol": "KNC",
         "decimals": 18,
-        "logoURI": "/images/main/empty-token.png"
```

Comment:
Better to not remove this logoURI prop from any of the tokens, because we are defaulting the logoURI with it even if is invalid because FE can default it with the default placeholder, removing it from here could lead to FE problems